__tests__/app.test.js

```diff
@@ -443,4 +443,17 @@ describe("GET /api/articles (topic query)", () => {
       expect(msg).toBe("Oops, does not exist yet")
     })
   });
+});
+
+describe('GET /api/articles/:article_id (comment_count)', () => {
+  it('should return article with comment count if query is set to true', () => {
+    return request(app).get("/api/articles/1?comment_count=true").expect(200).then(({body: {article}}) => {
+      expect(article).toHaveProperty("comment_count")
+    })
+  });
+  it('should return 400 Bad request if query is present but not allowed', () => {
+    return request(app).get("/api/articles/1?comment_count=roses").expect(400).then(({body: {msg}}) => {
+      expect(msg).toBe("Bad request")
+    })
+  });
```

Comment:
This should not be a query but a new property added to the base endpoint of `/api/articles/1` and within this url the response object should now have a new key of comment_count that is a number